src/helpers/lib/token.js

```diff
@@ -0,0 +1,182 @@
+exports.ABI = [
+  {
+    "constant":true,
+    "inputs":[
+
+    ],
+    "name":"name",
+    "outputs":[
+      {
+        "name":"",
+        "type":"string"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"view",
+    "type":"function"
+  },
+  {
+    "constant":false,
+    "inputs":[
+      {
+        "name":"_spender",
+        "type":"address"
+      },
+      {
+        "name":"_value",
+        "type":"uint256"
+      }
+    ],
+    "name":"approve",
+    "outputs":[
+      {
+        "name":"",
+        "type":"bool"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"nonpayable",
+    "type":"function"
+  },
+  {
+    "constant":true,
+    "inputs":[
+
+    ],
+    "name":"totalSupply",
+    "outputs":[
+      {
+        "name":"",
+        "type":"uint256"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"view",
+    "type":"function"
+  },
+  {
+    "constant":false,
+    "inputs":[
+      {
+        "name":"_from",
+        "type":"address"
+      },
+      {
+        "name":"_to",
+        "type":"address"
+      },
+      {
+        "name":"_value",
+        "type":"uint256"
+      }
+    ],
+    "name":"transferFrom",
+    "outputs":[
+      {
+        "name":"",
+        "type":"bool"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"nonpayable",
+    "type":"function"
+  },
+  {
+    "constant":true,
+    "inputs":[
+
+    ],
+    "name":"decimals",
+    "outputs":[
+      {
+        "name":"",
+        "type":"uint8"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"view",
+    "type":"function"
+  },
+  {
+    "constant":true,
+    "inputs":[
+      {
+        "name":"_owner",
+        "type":"address"
+      }
+    ],
+    "name":"balanceOf",
+    "outputs":[
+      {
+        "name":"balance",
+        "type":"uint256"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"view",
+    "type":"function"
+  },
+  {
+    "constant":true,
+    "inputs":[
+
+    ],
+    "name":"symbol",
+    "outputs":[
+      {
+        "name":"",
+        "type":"string"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"view",
+    "type":"function"
+  },
+  {
+    "constant":false,
+    "inputs":[
+      {
+        "name":"_to",
+        "type":"address"
+      },
+      {
+        "name":"_value",
+        "type":"uint256"
+      }
+    ],
+    "name":"transfer",
+    "outputs":[
+      {
+        "name":"",
+        "type":"bool"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"nonpayable",
+    "type":"function"
+  },
+  {
+    "constant":true,
+    "inputs":[
+      {
+        "name":"_owner",
+        "type":"address"
+      },
+      {
+        "name":"_spender",
+        "type":"address"
+      }
+    ],
+    "name":"allowance",
+    "outputs":[
+      {
+        "name":"",
+        "type":"uint256"
+      }
+    ],
+    "payable":false,
+    "stateMutability":"view",
+    "type":"function"
+  },
```

Comment:
Given that we only use the `.decimals()` and `.symbol()` parts of the token, we could strip this down into just those.

See the [Finance app](https://github.com/aragon/aragon-apps/tree/master/apps/finance/app/src/abi) as an example; it just creates an object out of those and gives it to `web3.eth.Contract()` as the ABI.